.github/workflows/build_package.yml

```diff
@@ -172,6 +172,7 @@ jobs:
           packages: "iree-runtime"
           output_dir: "${{ github.workspace }}/bindist"
         run: |
+          [ -e ./bindist/* ] && rm ./bindist/*
           ./c/build_tools/python_deploy/build_linux_packages.sh
```

Comment:
This feels like a generally good change for safety, though I am wondering: which files were left over that this will delete?

We do run a `clean_wheels` step already: https://github.com/openxla/iree/blob/b4251e2a3ee8b5809c00c055013e791ea48bf8a6/build_tools/python_deploy/build_linux_packages.sh#L121-L126 https://github.com/openxla/iree/blob/b4251e2a3ee8b5809c00c055013e791ea48bf8a6/build_tools/python_deploy/build_linux_packages.sh#L169-L174

Reply:
I _think_ the case is when we interrupt a build and we haven't hit a clean_wheels() call yet. It shows up like this release here: https://github.com/nod-ai/SRT/releases/tag/candidate-20230815.222 where another older release gets uploaded.

Reply:
Huh... sketchy. Could use a temp directory or encode the run id in the path (if the path length doesn't grow too large). Any directory cleanup like this on a persistent runner seems prone to race conditions otherwise...

Reply:
yeah then the temp dir keeps to grow (when we have lots of builds). This is a big hammer but cleans it out and wont have a race since we clear just before starting the job.